.gitignore

```diff
@@ -1,6 +1,36 @@
+*.project
+*.settings
+*.classpath
+*.metadata
+*.iml
 *.idea
+*.checkstyle
+*.DS_Store
+.history
+jsconfig.json
+.vscode/
+node_modules/
+node/
+atlassian-ide-plugin.xml
+*.sonar-ide.properties
+**/webapp/css/styles.css
+platform/solr/**/overlays
```

Comment:
✏️ Are some these lines from DDF? Are they necessary?
- *.checkstyle
- *.DS_Store
- .history
- jsconfig.json
- node_modules/
- node/
- atlassian-ide-plugin.xml
- *.sonar-ide.properties
- **/webapp/css/styles.css
- platform/solr/**/overlays

Reply:
Good question, probably not 🤷‍♂ I'll remove them and only add in the stuff that's needed for this PR